scss/footer.scss

```diff
@@ -0,0 +1,27 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding: 30px;
+  position: relative;
+
+  &::before {
+    content: url('assets/wolox_logo_white.svg');
+    position: relative;
+    padding: 10px;
```

Comment:
you don't want padding here, this should be a margin bottom, the top padding should be added directly to the parent